### PR TITLE
[SentinelOne] Update argument computer_name of command sentinelone-list-agents

### PR DESCRIPTION
**Pull request by chloerongier**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This PR updates the argument *computer_name* so the **sentinelone-list-agents** command  will now be able to match partial computer names value (substring) which was not possible before.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by kgal-pan** [comment]
@munna-metron @parag-metron @saurabh-metron - Can you please review and approve these changes?

**Comment by munna-metron** [comment]
> @munna-metron @parag-metron @saurabh-metron - Can you please review and approve these changes?

HI @kgal-pan , reviewed the changes and those are looking good. Thanks
cc: @parag-metron @saurabh-metron

File: Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml [hunk 1]
```diff
@@ -130,7 +130,7 @@ name: SentinelOne V2
 script:
   commands:
   - arguments:
-    - description: The computer name by which to filter the results.
+    - description: The computer name by which to filter the results. It can match a partial computer name value (substring).
```
Comment (YuvHayun) [inline Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml:133]: Make sure to edit the readme as well.

**Review by YuvHayun — CHANGES_REQUESTED** [review]
Looks good, added 1 small comment.
Just to make sure **computerName__like** is as shown in the endpoint docs?
can you add a link to it please?
At the meantime, triggering instance test.

**Comment by YuvHayun** [comment]
@chloerongier please also make sure you're up to date with master.

**Comment by chloerongier** [comment]
> Looks good, added 1 small comment. Just to make sure **computerName__like** is as shown in the endpoint docs? can you add a link to it please? At the meantime, triggering instance test.

Here is the URI to the doc "/api-doc/api-details?category=agents&api=get-agents" and a screenshot with the query params :
image

**Comment by YuvHayun** [comment]
> > Looks good, added 1 small comment. Just to make sure **computerName__like** is as shown in the endpoint docs? can you add a link to it please? At the meantime, triggering instance test.
> 
> Here is the URI to the doc "/api-doc/api-details?category=agents&api=get-agents" and a screenshot with the query params : image

Thx!
So please just pull from master.

**Comment by YuvHayun** [comment]
Merging to internal PR.
Thx for your contribution @chloerongier

**Pull request merged**